pkg/aggregator/aggregator.go

```diff
@@ -765,8 +842,19 @@ func (agg *BufferedAggregator) run() {
 					log.Errorf("Error submitting orchestrator data: %s", err)
 				}
 			}(orchestratorMetadata)
-		}
+		case event := <-agg.eventPlatformIn:
+			state := stateOk
+			aggregatorEventPlatformEvents.Add(event.eventType, 1)
+			err := agg.handleEventPlatformEvent(event)
+			if err != nil {
+				state = stateError
+				aggregatorEventPlatformLatestError = err
+				aggregatorEventPlatformEventsErrors.Add(event.eventType, 1)
+				log.Debugf("error submitting event platform event: %s", err)
```

Comment:
This should be an `Errorf`, no ?

Reply:
No, that's intended to be a debug. This is related to your other comment https://github.com/DataDog/datadog-agent/pull/7735#discussion_r612594808 

My intention here is to avoid spamming the logs by default if there's a problem. So if the event type is wrong, or the downstream pipeline is backed up, we'll log an error message only once per aggregator flush interval, in `logPostFlushMessages`. If a check is submitting 1k events per second then logging one error per event would be excessive, which is why I figured that should only happen if debug is enabled.

One error message per flush interval should be enough to indicate there's a problem, at which point the user could look at telemetry and/or enable debug logs for more info. 

Do we have a better way to show errors in a way that we avoid avoid excessively spamming the logs?

Reply:
I'm still missing where we reset `aggregatorEventPlatformLatestError`. Unless I'm missing something obvious, once an error happend we will log the same error on every loop.

Reply:
Oh good point. I had a reset earlier then must have removed it during some refactoring. 

> Also what would you think about logging the error right when it happened and then setting a bool to true to avoid logging the next ones until we get here

Good idea, that's better. Updated here https://github.com/DataDog/datadog-agent/pull/7735/commits/1d4fd01c28a23ec0f5d0530dce6629d6634d53a4